### PR TITLE
Resolves #2735

### LINKED ISSUE #2735: Bad parameter description for the AllowDeviceAuthAsPrimaryForDomainJoinedDevices and ClientAuthenticationMethods parameters
In the official page, you can see that you have the description of "AdditionnalAuthenticationProvider" for the 2 next AllowDeviceAuthAsPrimaryForDomainJoinedDevices and ClientAuthenticationMethods parameters. It's a bad copy, but we don't know what is the definition of those parameters.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5c5337c9-c9f1-bcde-6594-56fe86d92b9f
* Version Independent ID: cadb3cc7-3d88-ae91-f244-7b516e2c066d
* Content: [Set-AdfsGlobalAuthenticationPolicy (ADFS)](https://docs.microsoft.com/en-us/powershell/module/adfs/set-adfsglobalauthenticationpolicy?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

### PR DESCRIPTION
**Pull request by velkovb**
Update parameter description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2735

### CODE REVIEW

File: docset/winserver2016-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md [hunk 1]
```diff
@@ -67,8 +67,7 @@ Accept wildcard characters: False
 ```
 
 ### -AllowDeviceAuthAsPrimaryForDomainJoinedDevices
-Specifying this parameter configures an external authentication provider, for second stage authentication, in the global policy.
-This is the first step in creating an AD FS policy that invokes an external authentication provider for multifactor authentication.
+Allows to use device authentication as primary for domain joined devices.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md:70]:
```suggestion
Allows the use of device authentication as primary authentication for domain-joined devices.
```

File: docset/winserver2019-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md [hunk 2]
```diff
@@ -67,8 +67,7 @@ Accept wildcard characters: False
 ```
 
 ### -AllowDeviceAuthAsPrimaryForDomainJoinedDevices
-Specifying this parameter configures an external authentication provider, for second stage authentication, in the global policy.
-This is the first step in creating an AD FS policy that invokes an external authentication provider for multifactor authentication.
+Allows to use device authentication as primary for domain joined devices.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md:70]:
```suggestion
Allows the use of device authentication as primary authentication for domain-joined devices.

```

File: docset/winserver2022-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md [hunk 3]
```diff
@@ -67,8 +67,7 @@ Accept wildcard characters: False
 ```
 
 ### -AllowDeviceAuthAsPrimaryForDomainJoinedDevices
-Specifying this parameter configures an external authentication provider, for second stage authentication, in the global policy.
-This is the first step in creating an AD FS policy that invokes an external authentication provider for multifactor authentication.
+Allows to use device authentication as primary for domain joined devices.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md:70]:
```suggestion
Allows the use of device authentication as primary authentication for domain-joined devices.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by JohanFreelancer9** [comment]
Apologies Borislav, I've also added the word "method", could you please update? Many thanks.

**Comment by velkovb** [comment]
Not sure if method is the correct word here as it may create confusion. Both in Client and Device parameter method is used to specify the authentication method. Maybe here it is more appropriate to say type?

File: docset/winserver2016-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md [hunk 4]
```diff
@@ -67,8 +67,7 @@ Accept wildcard characters: False
 ```
 
 ### -AllowDeviceAuthAsPrimaryForDomainJoinedDevices
-Specifying this parameter configures an external authentication provider, for second stage authentication, in the global policy.
-This is the first step in creating an AD FS policy that invokes an external authentication provider for multifactor authentication.
+Allows the use of device authentication as primary authentication method for domain-joined devices.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md:70]:
```suggestion
Allows the use of device authentication as the primary type for domain-joined devices.
```

File: docset/winserver2019-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md [hunk 5]
```diff
@@ -67,8 +67,8 @@ Accept wildcard characters: False
 ```
 
 ### -AllowDeviceAuthAsPrimaryForDomainJoinedDevices
-Specifying this parameter configures an external authentication provider, for second stage authentication, in the global policy.
-This is the first step in creating an AD FS policy that invokes an external authentication provider for multifactor authentication.
+Allows the use of device authentication as primary authentication for domain-joined devices.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md:70]:
```suggestion
Allows the use of device authentication as the primary type for domain-joined devices.
```

File: docset/winserver2022-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md [hunk 6]
```diff
@@ -67,8 +67,8 @@ Accept wildcard characters: False
 ```
 
 ### -AllowDeviceAuthAsPrimaryForDomainJoinedDevices
-Specifying this parameter configures an external authentication provider, for second stage authentication, in the global policy.
-This is the first step in creating an AD FS policy that invokes an external authentication provider for multifactor authentication.
+Allows the use of device authentication as primary authentication for domain-joined devices.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adfs/Set-AdfsGlobalAuthenticationPolicy.md:70]:
```suggestion
Allows the use of device authentication as the primary type for domain-joined devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@velkovb, please commit the changes if you agree. Thanks for the explanation.

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**